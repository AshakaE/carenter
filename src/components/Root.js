import React from 'react';
import { Link } from 'react-router-dom';
import styles from '../assets/css/home.module.css';
import carIndex from '../assets/img/carIndex.png';

const Root = () => {
  const one = 1; //eslint-disable-line

  return (
    <>
      <div>
        <img src={carIndex} alt="car" className={styles.img} />
        <div className={styles.container}>
          <div className={styles.items}>
            <div className={styles.header}>
              <h1>Welcome to Carenter</h1>
            </div>
            <div className={styles.btn}>
              <Link to="/login-or-signup">Book a ride now</Link>
            </div>
          </div>
        </div>
      </div>
    </>
  );
};

export default Root;
