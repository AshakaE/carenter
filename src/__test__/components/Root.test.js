import React from 'react';
import { BrowserRouter } from 'react-router-dom';
import renderer from 'react-test-renderer';
import Root from '../../components/Root';

describe('Root', () => {
  it('renders correctly', () => {
    const tree = renderer
      .create(
        <BrowserRouter>
          <Root />
        </BrowserRouter>,
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
