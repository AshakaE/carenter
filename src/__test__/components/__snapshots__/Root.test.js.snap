// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Root renders correctly 1`] = `
<div>
  <img
    alt="car"
    className="img"
    src="carIndex.png"
  />
  <div
    className="container"
  >
    <div
      className="items"
    >
      <div
        className="header"
      >
        <h1>
          Welcome to Carenter
        </h1>
      </div>
      <div
        className="btn"
      >
        <a
          href="/login-or-signup"
          onClick={[Function]}
        >
          Book a ride now
        </a>
      </div>
    </div>
  </div>
</div>
`;
